sdk/ml/azure-ai-ml/azure/ai/ml/operations/_online_deployment_operations.py

```diff
@@ -184,7 +184,8 @@ def begin_create_or_update(
             upload_dependencies(deployment, orchestrators)
             try:
                 location = self._get_workspace_location()
-                if kwargs.pop("package_model", False):
+                is_package_model = deployment.package_model if hasattr(deployment, "package_model") else False
```

Comment:
Make sure it does not break the CLI , https://msdata.visualstudio.com/DefaultCollection/Vienna/_git/sdk-cli-v2?path=/src/cli/src/machinelearningservices/azext_mlv2/manual/custom/online_deployment.py&version=GBmain&line=116&lineEnd=120&lineStartColumn=1&lineEndColumn=10&lineStyle=plain&_a=contents , so having CLI tests successfull is important